src/XHRConnection.js

```diff
@@ -128,11 +128,17 @@ export default class XHRConnection extends AbstractConnection {
   /**
    * create, prepare, open and send the xhr request
    */
-  open() {
+  open(options = {}) {
```

Comment:
Why not handle it in the constructor like the other options?

Reply:
it won't really matter since it's going to be hidden behind http-service

Reply:
though I think that could be another option

Reply:
With it being an option on open, I'm not sure how we could get that thru the connectionmanager and what not?

Reply:
fixed